tests/webpacker/webpack.schema.json

```diff
@@ -0,0 +1,2062 @@
+{
```

Comment:
Let's import this file directly from the node module (`https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json`)